web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -364,20 +386,20 @@ describe("<BusinessFormationPaginator />", () => {
         expect(page.getStepStateInStepper(LookupStepIndexByName("Name"))).toEqual("COMPLETE");
       });
 
-      it("marks step one as incomplete if business name is unavailable", async () => {
+      it("marks step one as error if business name is unavailable", async () => {
         const page = preparePage(initialUserData, displayContent);
         page.fillText("Search business name", "Pizza Joint");
         await page.searchBusinessName({ status: "UNAVAILABLE" });
         switchStepFunction();
-        expect(page.getStepStateInStepper(LookupStepIndexByName("Name"))).toEqual("INCOMPLETE");
+        expect(page.getStepStateInStepper(LookupStepIndexByName("Name"))).toEqual("ERROR");
       });
 
-      it("marks step one as incomplete if business name search is error", async () => {
+      it("marks step one as error if business name search is error", async () => {
         const page = preparePage(initialUserData, displayContent);
         page.fillText("Search business name", "Pizza Joint LLC");
         await page.searchBusinessName({ status: "DESIGNATOR" });
```

Comment:
Would it be worthwhile to rename the DESIGNATOR status (or the test) so that it clearly reflects an error state? Both Soma and I had to trace the status back to `APIBusinessNameClient.ts`. We're thinking about self-contained code so thought this would be an example of code block we could change. Thoughts?

Reply:
Sure, what are you thinking?

Reply:
I was thinking just having status suffixed with the keyword error, so like `DESIGNATOR_ERROR` so if anyone were to read the test or anywhere else in the code it would be obvious it's a type of error.

Reply:
How about `it ("marks step one as error if business name search returns an error response status")`?